### ISSUE TITLE
[Feature]: AvatarStack

### ISSUE THREAD

**Issue by AndreaKarlsen** [HelseVestIKT/hviktor-blazor#30]
### Base example
- Wrapper missing role="group"
- Wrapper missing accessible name, add aria-label e.g. "Top 5 employees"
- Avatars in the group have empty aria-label attributes. Please add a value for a realistic example. 
<img width="1624" height="770" alt="Image" src="https://github.com/user-attachments/assets/30b265aa-2442-48b4-9788-4aec0074baee" />

Look at the [Angular example](https://helsevestikt.github.io/hviktor/komponenter/avatar-stack) for inspiration. 

### Customization
- You should not be able to have a overlap higher than 50. Higher values dont give the stack any value for the user. 
- Not necessary with this much flexibility. We should only have 0px, 4px and 8px (more in line with sizes we use)
- The size of the Avatar should only be available in the defined sizes small, medium and large. Small should however only be used on mobile. (The control is kinda buggy on the demo-page). 
- The group of checkboxes (expandable + square) is not coded as a fieldset with legend (We should think about accessibility also on the demo-page). You have added aria-describedby that refers to an ID that does not exist (both inputs). 

### Section about sizes
- As described above, we should only have the defined sizes of small, medium and large.
